pkg/registry/helm.go

```diff
@@ -54,7 +54,7 @@ func handleChart(chart *image.HelmChart, repo *image.HelmRepository, valuesDir,
 		return nil, fmt.Errorf("downloading chart: %w", err)
 	}
 
-	images, err := getChartContainerImages(chart, helmClient, chartPath, valuesPath, kubeVersion)
+	images, err := getChartContainerImages(chart, helmClient, chartPath, valuesPath, kubeVersion, chart.TargetNamespace)
```

Comment:
We already pass the full chart - there's no need to provide the namespace separately.